src/main/java/de/bonndan/nivio/config/SeedProperties.java

```diff
@@ -0,0 +1,19 @@
+package de.bonndan.nivio.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
+
+@ConfigurationProperties(prefix = "nivio")
+@Validated
```

Comment:
There is a @Validated annotation, but no further annotations saying what to validate. In case of SEED we have a string of one or more URLs or relative paths, so it's hard to validate.

Reply:
Okay, ich habe schon geändert